doc/rtd/reference/yaml_examples/apt.rst

```diff
@@ -0,0 +1,369 @@
+.. _cce-apt:
+
+Additional APT config and repos
```

Comment:
There's a looooot of explanation in this and the other apt page. Should we remove it all or move it into the module documentation?

Reply:
Most of the apt stuff (if not all) came from the example folder. I'm very happy to get rid of anything that's duplicated. The module docs seem like the most sensible place to document/explain the behaviour but I might need a hand to make sure I don't accidentally get rid of anything important.

Reply:
I've been through this old example, and everything that's represented here is already in the module docs - so I've just removed it all